app/models/provisioning_template.rb

```diff
@@ -190,4 +190,14 @@ def self.pxe_default_combos
     end
     combos
   end
+
+  def self.local_boot_template(host)
+    return unless host.operatingsystem.present?
```

Comment:
@dLobatog Do you mean putting an instance method for Host::Managed named `local_boot_template` and then referencing that inside `host.provisioning_template`?